app/scripts/controllers/specialErrorDetail.js

```diff
@@ -26,23 +28,48 @@ module.exports = /*@ngInject*/ function ($scope, $routeParams, $location, $http,
         $scope.error = {};
     }
 
+    $scope.sort = function(property) {
+        $scope.reverse = $scope.sortedBy === property ? !$scope.reverse : false;
```

Comment:
Ahh...so reverse is supposed to indicate the direction of the sort by inferring that it is either in a reverse sort or not. That's backwards. :(
If that's the case, then I think sortAsc makes the most sense.